app/models/library_staff_document.rb

```diff
@@ -82,7 +82,9 @@ def type
   def description; end
 
   def url
-    "https://library.princeton.edu/staff/#{document.netid}"
+    "https://library.psb-prod.princeton.edu/people/#{document.first_name}-#{document.last_name}"
```

Comment:
I think it would probably cover more eventualities if we just use the [URL encoding described here](https://dev.to/bdavidxyz/how-to-encode-an-url-string-in-ruby-2g) (e.g. apostrophes, accents, etc.)

Reply:
Do you think it's necessary to handle the question mark edge case it describes or just run it through the URI Parser?

Reply:
I think it would be fine to just run it through the URI parser